cuqi/data/_data.py

```diff
@@ -315,7 +316,40 @@ def p_power(size=128, relnz=0.3, p=2, seed=1): #relnz=0.65, p=2.3
     
     return x
 
+def cookie(size=128, grayscale=True):
+    """ Cartoon-style image of a cookie.
 
+    Parameters
+    ----------
+    size : int
+        Size of the image to generate. Image is square with sides of length size.
```

Comment:
COuld you please state somewhere around here
- that the image is generated by loading it in from a PNG file
- what the original size of the image in the PNG is (so that if one wants to avoid resampling then one knows what size to use
- consider if the default size should be the non-resampled size

Reply:
Done